### PR TITLE
Apply minor styling updates

### PR DESCRIPTION
**Pull request by mattdsmith89**

### CODE REVIEW

**Comment by mattdsmith89** [comment]
Much white space updates, stupid auto-format 😞

File: src/Piston/Views/Post/_Preview.cshtml [hunk 1]
```diff
@@ -1,12 +1,16 @@
-﻿@model Piston.Models.Post
+﻿<div class="row clear">
+    <div class="col-12 col-10-m col-8-l">
+        @model Piston.Models.Post
```
Comment (daveraine) [inline src/Piston/Views/Post/_Preview.cshtml]: Why this not first line? :confused:

File: src/Piston/scss/base.scss [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
 
+// Override base's default variables
+$base-font-family: 'Bitter', serif; 
+$base-heading-font-family: 'Raleway', sans-serif;
+$base-line-height: 30;
+$base-link-color: black;
+$base-link-hover-color: #808080;
+$banner-background-color: black;
+$brand-font-color: white;
+$banner-font-color: white;
```
Comment (daveraine) [inline src/Piston/scss/base.scss]: Overrides go in scss/styles/_variables-colors.scss yo :sunglasses:

**Review by daveraine — CHANGES_REQUESTED** [review]

File: src/Piston/Views/Post/_Preview.cshtml (hunk 1 above)
Comment (mattdsmith89) [reply src/Piston/Views/Post/_Preview.cshtml]: Because I wasn't paying attention 😄

File: src/Piston/scss/base.scss (hunk 2 above)
Comment (mattdsmith89) [reply src/Piston/scss/base.scss]: I did look to see if you had overrides somewhere, that wasn't at all obvious! 😛

Comment (daveraine) [reply src/Piston/scss/base.scss]: It was the default folder structure from the base download so I left it. I think we should move all the ad-hoc styling from site.scss into individual files to make it easier to manage, will make an issue for that.

**Comment by daveraine** [comment]
Here's what it looks like now:

home

post

**Comment by mattdsmith89** [comment]
It's not perfect by all means, I was really just making it look less like the base.. base style. 

There are still plenty of issues with it:
- the HRs look weird on the main page
- review text might be a bit too wide
- score looks a bit off
- when the scroll bar appears, everything shifts left a bit which looks naff
- review byline doesn't match front page (should it?)

I'm sure there are many more.

Should probably open an issue (or many issues) about styling the site properly really.

**Comment by Kitcheone** [comment]
:+1:

**Comment by daveraine** [comment]
Ok. I suggest you make issues for those points (or combine into one issue, whatevs) and we will merge this. I think it's a good start, maybe needs more colour. Might need links in the header at some point. You got any comments @Kitcheone?

**Review by daveraine — APPROVED** [review]

**Comment by mattdsmith89** [comment]
Yeah I agree with that. It's definitely lacking colour, but I think reviews should probably have images too and that would help. Also agree with links in the header - just need to be careful when viewing the site on a small device. Don't think this "framework" will auto-collapse the header menu like bootstrap does. Would get hideously messy on mobile if we don't implement something like that.

**Comment by Kitcheone** [comment]
Yeah I think it needs colour and a lot more work, but it's already better than base base.

Put those issues somewhere - github project for front end?

**Comment by mattdsmith89** [comment]
Opened issue #8 to cover this for now.

**Pull request merged**